src/bookshelf/src/bksw_conf.erl

```diff
@@ -97,14 +109,11 @@ ends_with(Char, String) ->
 %%%===================================================================
 %%% Internal functions
 %%%===================================================================
+auth_check_disabled() ->
+    envy:get(bookshelf, auth_check_disabled, false, boolean).
 
 ip() ->
-    case application:get_env(bookshelf, ip) of
-        undefined ->
-            [{ip, "127.0.0.1"}];
-        {ok, Ip} ->
-            [{ip, Ip}]
-    end.
+    {ip, envy:get(bookshelf, ip, "127.0.0.1", string)}.
```

Comment:
We used to return `[{ip, Ip}]` and now we're returning `{ip, Ip}`.

1. Is this a bug
2. If not, should we make a `spec()` for this?